### PR TITLE
fix: tabs directional arrow key bug

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
We were overthinking direction in tabs ;). No need for direction check.

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Review by awentzel — APPROVED** [review]

File: packages/web-components/fast-foundation/src/tabs/tabs.ts [hunk 1]
```diff
@@ -242,11 +239,11 @@ export class Tabs extends FASTElement {
             switch (keyCode) {
                 case keyCodeArrowLeft:
                     event.preventDefault();
-                    this.adjust(this.direction === Direction.rtl ? 1 : -1);
+                    this.adjust(1);
```
Comment (SethDonohue) [inline packages/web-components/fast-foundation/src/tabs/tabs.ts:242]: This is backwards I believe. Arrow Key Left should adjust -1.

File: packages/web-components/fast-foundation/src/tabs/tabs.ts [hunk 2]
```diff
@@ -242,11 +239,11 @@ export class Tabs extends FASTElement {
...
                     break;
                 case keyCodeArrowRight:
                     event.preventDefault();
-                    this.adjust(this.direction === Direction.rtl ? -1 : 1);
+                    this.adjust(-1);
```
Comment (SethDonohue) [inline packages/web-components/fast-foundation/src/tabs/tabs.ts:246]: Similar comment as above, but arrow key right should +1.

**Review by SethDonohue — COMMENTED** [review]
Am I missing something here? I tested this PR and arrow key left moves the tabs to the right, opposite is true for the arrow key right...

Any why do we not need the Direction check and switch?

File: packages/web-components/fast-foundation/src/tabs/tabs.ts (hunk 1 above)
Comment (eljefe223) [reply packages/web-components/fast-foundation/src/tabs/tabs.ts:242]: your right! ;)

**Review by nicholasrice — APPROVED** [review]

**Pull request merged**